src/ast_walk_interpreter.rs

```diff
@@ -316,12 +316,7 @@ pub fn call_func(func: &Function, arg_vals: &[Value]) -> Result<Option<Value>, R
             Ok(None)
         }
         Function::NativeReturning(_, ref native_fn) => Ok(Some(native_fn(arg_vals.to_vec())?)),
-        Function::User {
-            ref param_names,
-            ref body,
-            ref env,
-            ..
-        } => {
+        Function::User { ref param_names, ref body, ref env, .. } => {
```

Comment:
That's odd. The rustfmt lint is failing on Travis is failing too.

I _think_ this is your rustfmt version. Mine (and Travis') is currently 0.8.3.